spec/models/card_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+describe Card do
+  let(:card) {Card.create(original_text: "hausE SOmE ДоМ", translated_text: "hauses")}
+  let(:card_bad) {Card.new(original_text: "home", translated_text: "home", review_date: Date.today)}
+
+  it 'check translation' do
+    expect(card.check_translation("вигвам")).to be false
+  end
+
+  it 'set review_date in fields with the same values not work' do
+    expect(card_bad.update_review_date!).to be false
+  end
+
+  it 'should check correct translation with strange case' do
+    expect(card.check_translation("hausE SOmE ДоМ")).to be true
+  end
+
+  it 'should set corrected review_date' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.